sdk/search/ci.mgmt.yml

```diff
@@ -15,9 +15,7 @@ trigger:
     include:
       - sdk/search/
     exclude:
-      - sdk/search/azure
-      - sdk/search/microsoft-azure
-      - sdk/search/ms-azure
+      - sdk/search/azure-search
```

Comment:
Search rename the model to azure-search-documents

Reply:
Fixed.